DevelopmentProposals/reporterColumnLabels/reporterColumnLabels.md

```diff
@@ -54,14 +54,15 @@ and forcing the user to specify an annotation for each Output may not be practic
 - The TableReporter should use a default column name that is very likely to be meaningful.
 
 ### Architecture
-The selected design involves four main changes:
+The selected design involves six main changes:
 
 1. Rename "annotation" to "alias" (e.g., `AbstractInput::getAnnotation()` becomes `AbstractInput::getAlias()`).
 2. If the user provides the optional second argument when connecting an Output to an Input, store the string as the Input's alias; otherwise, the alias remains null.
-3. Create `getShortLabel()` and `getLongLabel()` methods on Input:
-  - `getShortLabel()` will be called by ConsoleReporter and will return `isNull(alias) ? output_name : alias`.
-  - `getLongLabel()` will be called by TableReporter and will return `isNull(alias) ? full_path_name : alias`.
-4. Create `setAlias()` methods analogous to the current `getAnnotation()` methods so the user can change the alias after (but not before) connecting.
+3. Create `getReportedLabel()` on Input, which will always return the alias if one is provided:
+  - `getReportedLabel()` will be called by a Reporter and will return `isNull(alias) ? full_path_name : alias`.
```

Comment:
> as we discussed, there may be other consumers that will want a label, with a preference for a short or long version

Why would a user not request the *alias* directly that they assigned and use either `getShortName()` or `getLongName()` which will give the same value if an alias is provided? What are the non Reporter use cases for  `getShortLabel()` and `getLongLabel()`? Who else needs to truncate other than the `ConsoleReporter`? That issue was punted to `getShortLabel()`. 

My interpretation of the meeting was that short and long name were used to refer to the alias or output name and alias or full_path_name in the discussion and not that they would be the actual method names.  When I viewed the revised DevProp I thought the names were stand ins to be revised. I was surprised when the DevProp got merged today without review and that you had a PR in.

I'm very sorry for taking a weekend to mull it over. If you believe that I raise valid points then I will make the changes so your time is not wasted. Otherwise, leave it as is and I will kill this PR.